athena-federation-integ-test/pom.xml

```diff
@@ -82,6 +82,12 @@
             <artifactId>aws-java-sdk-cloudformation</artifactId>
             <version>${aws-sdk.version}</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-ec2 -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-ec2</artifactId>
+            <version>${aws-sdk.version}</version>
+        </dependency>
```

Comment:
Is this to support the new VPC info? and can you remind me if the appropriately setup the integ-test pom and dependencie such that they don't end up in the Lambda jar?

Reply:
this will be removed when I switch the VPC logic in favor of the config file. Moot point, but these dependencies are not in the Lambda jar since we are including the integ-test module as a test dependency.